index.html

```diff
@@ -22,22 +25,23 @@
           </div>
 
           <div class="pre-navigation-links">
-            <h5><a>English</a></h5>
-            <h5><a>My Page</a></h5>
-            <h5><a>Logout</a></h5>
+            <h5><a class="pre">English</a></h5>
+            <h5><a class="pre">My Page</a></h5>
+            <h5><a class="pre">Logout</a></h5>
           </div>
         </div>
       </div>
       <div class="nav">
-        <div id="hamburger" class="hamburger">
-          <span class="bar"></span>
-          <span class="bar"></span>
-          <span class="bar"></span>
-        </div>
+				<a href="index.html"><img src="/Capstone-One/imgs/logo-orange.png" class="logo-nav" alt="logoporcilio"></a>
```

Comment:
[OPTIONAL] All the images on your web pages are broken (Logo, speakers, program, partners ... ). I recommend fixing the paths in the src attribute. For example, at this line, the src should be `imgs/logo-orange.png` instead of `/Capstone-One/imgs/logo-orange.png`.

Reply:
Hi Amine!,

Thanks for all your comments. But in this case in particular I don't have that problem. May be the pics are no load in your phone yet.!


[Screenshot_20220805-160851_Chrome.jpg](https://user-images.githubusercontent.com/99001924/183145038-cd339e94-8807-40ec-96b3-c1d0c637e497.jpg)

Reply:
Hi Marco, I used my local machine and the live server extension to open your web pages.

<img width="642" alt="image" src="https://user-images.githubusercontent.com/77190203/183149042-e1d9ef0d-dc66-4eee-a5d5-fa9a31a1807e.png">

Reply:
I use not only live server, use several phones y desktop with Live Demo link, and other partners did the same to verify, and no one have that problem... making those changes in the rute of src, ill make it more universally to any computer? what could be the reason of the issue?